tests/installation/logs_from_installation_system.pm

```diff
@@ -56,6 +57,9 @@ sub run {
     # We don't change network setup here, so should work
     # We don't parse logs unless it's detect_yast2_failures scenario
     $self->save_upload_y2logs(no_ntwrk_recovery => 1, skip_logs_investigation => !get_var('ASSERT_Y2LOGS'));
+
+    # Change setting as 15-sp2 don't need set username but 11-SP4 need set username
+    set_var('DM_NEEDS_USERNAME', '0') if (check_var('MEDIA_UPGRADE', '1') && is_sle('=11-SP4', get_var('HDDVERSION')));
```

Comment:
We'd better to judge if the variable eq 1 before set it to 0. Besides, if consider automate the 11SP4 on s390x z/VM, maybe it is not very suitable to set it to 0 on the preparation of such test.